charts/kapacitor/templates/deployment.yaml

```
@@ -12,6 +12,8 @@ metadata:
     app: {{ template "kapacitor.fullname" . }}
 spec:
   replicas: 1
+  strategy:
+{{ toYaml .Values.strategy | trim | indent 4 }}
```

With `strategy` commented out in `values.yaml` by default, the template output is 
```
  strategy:
    null
```
I guess it should be change to
```suggestion
  {{- if .Values.strategy }}
  strategy:
{{ toYaml .Values.strategy | trim | indent 4 }}
  {{- end }}
```

---

@iblackman can you address this? then we are ready to merge